boskos/ranch/ranch.go

```diff
@@ -127,6 +128,7 @@ func (r *Ranch) Acquire(rType, state, dest, owner string) (*common.Resource, err
 // In: state - current state of the requested resource
 //     dest - destination state of the requested resource
 //     owner - requester of the resource
+//     names - names of resources to acquire
```

Comment:
nit: names of resource types to acquire

Reply:
no its resource name, not type.

Reply:
removed the "s"

Reply:
ohhh humm then I understand this wrong - gonna walk this through again